components/profile/ProfileDetail.vue

```diff
@@ -170,19 +174,30 @@ import CollectionGrid from '@/components/collection/CollectionGrid.vue'
 import Activity from './activityTab/Activity.vue'
 import { useListingCartStore } from '@/stores/listingCart'
 
+enum ProfileTab {
+  OWNED = 'owned',
+  CREATED = 'created',
+  COLLECTIONS = 'collections',
+  ACTIVITY = 'activity',
+}
+
 const route = useRoute()
 const { toast } = useToast()
 const { replaceUrl } = useReplaceUrl()
 const { accountId } = useAuth()
 const { urlPrefix } = usePrefix()
 const listingCartStore = useListingCartStore()
 
-const tabs = ['owned', 'created', 'collections', 'activity']
+const itemTabs = [ProfileTab.OWNED, ProfileTab.CREATED]
```

Comment:
this and `showItemTabs` are not very good names
maybe just 'spread' it out in the template (i.e. no loop)
or be creative and think of a better name